contracts/CurrencyNetwork.sol

```diff
@@ -267,6 +268,48 @@ contract CurrencyNetwork is CurrencyNetworkInterface, Ownable, Authorizable, Des
         }
     }
 
+    /**
+     * @notice send `_value` token to `_to` from `_from`
+     * `_from` needs to have a debt towards è_to` of at least `_value`
+     * `_to` needs to be msg.sender
+     * @param _from The address of the sender
+     * @param _to The address of the recipient
+     * @param _value The amount of token to be transferred
+     * @param _maxFee Maximum fee the sender wants to pay
+     * @param _path Path between _from and _to
+     * @param _extraData extra data bytes to be logged in the Transfer event
+     **/
+    function debitTransfer(
+        address _from,
+        address _to,
+        uint64 _value,
+        uint64 _maxFee,
+        address[] calldata _path,
+        bytes calldata _extraData
+    )
+        external returns (bool success)
+    {
+        require(_to == msg.sender, "The transfer can only be initiated by the creditor (_to).");
+        require(getDebt(_from, _to) >= _value, "The sender does not have such debt towards the receiver.");
+
+        success = _mediatedTransferReceiverPays(
+            _from,
+            _to,
+            _value,
+            _maxFee,
+            _path);
+
+        if (success) {
+            emit Transfer(
+                _from,
+                _to,
+                _value,
+                _extraData
+            );
+            _reduceDebt(_from, _to, _value);
```

Comment:
We should first do this. We dont call external stuff at the moment, but I would rather still follow that pattern. 
We can just revert on success false.